pkgs/top-level/release-cross.nix

```diff
@@ -89,101 +89,26 @@ in
     guile = nativePlatforms;
   };
 
-  darwinToAarch64 = let
-    crossSystem = {
-      config = "aarch64-apple-darwin14";
-      arch = "arm64";
-      libc = "libSystem";
-    };
-  in mapTestOnCross crossSystem darwinCommon;
+  darwinToAarch64 = mapTestOnCross lib.systems.examples.iphone64 darwinCommon;
 
-  darwinToArm = let
-    crossSystem = {
-      config = "arm-apple-darwin10";
-      arch = "armv7-a";
-      libc = "libSystem";
-    };
-  in mapTestOnCross crossSystem darwinCommon;
+  darwinToArm = mapTestOnCross lib.systems.examples.iphone32 darwinCommon;
```

Comment:
Oh I didn't realize you were commenting on the name that already existed. It's purely "from Darwin" only as I haven't generalized it yet, whereas the others are from Linux.

Eventually the build system shouldn't matter for most things, but not there yet.